plugin/code_actions.py

```diff
@@ -334,4 +307,12 @@ def show_popup_menu(self) -> None:
     def handle_select(self, index: int) -> None:
         if index > -1:
             selected = self.commands[index]
-            run_code_action_or_command(self.view, selected[0], selected[2])
+            session = self.session_by_name(selected[0])
+            if session:
+
+                def print_to_status_bar(_: Any) -> None:
+                    window = self.view.window()
+                    if window:
+                        window.status_message('Applied "{}"'.format(selected[1]))
```

Comment:
Grr. :)

- During code-actions-on-save this just causes a flash that is impossible to read because it's immediately replaced by "file was saved" message.
- During manually applying code action I don't see much point of it as I know which code action I have just selected and don't really need confirmation that all went well.